scss/civicrm/financial/_financial-batch.scss

```diff
@@ -0,0 +1,29 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#{civi-page('financial-batch')} {
+  .crm-container {
```

Comment:
why is this necessary? We are adding `.crm-container` to every selector when we build the css file